packages/@angular/cli/models/webpack-configs/typescript.ts

```diff
@@ -9,6 +10,28 @@ const webpackLoader: string = g['angularCliIsLocal']
   : '@ngtools/webpack';
 
 
+function _createAotPlugin(wco: WebpackConfigOptions, options: any) {
+  const { appConfig, projectRoot, buildOptions } = wco;
+
+  // Read the environment, and set it in the compiler host.
+  const appRoot = path.resolve(projectRoot, wco.appConfig.root);
```

Comment:
Shouldn't `wco.appConfig.root` be `appConfig.root` since it was already destructured from wco? (same on the next 2 lines too)